Cohort2/team4/client/src/components/NavBar.tsx

```diff
@@ -1,32 +1,47 @@
                     <span className="text-xl text-black">Convi</span>
+                    </a>
                 </div>
                 <div className="w-full block flex-grow lg:flex lg:items-center lg:w-auto">
                     <div className="text-sm lg:flex-grow">
-                        <a href="#responsive-header" className="block mt-4 lg:inline-block lg:mt-0  text-black-200 font-medium hover:text-blue-800 mr-4">
+                        <a href="#" className="block mt-4 lg:inline-block lg:mt-0  text-black-200 font-medium hover:text-blue-800 mr-4">
                             About Us
                         </a>
-                        <a href="#responsive-header" className="block mt-4 lg:inline-block lg:mt-0 text-black-200 font-medium hover:text-blue-800 mr-4">
+                        <a href="#" className="block mt-4 lg:inline-block lg:mt-0 text-black-200 font-medium hover:text-blue-800 mr-4">
                             Features
                         </a>
-                        <a href="#responsive-header" className="block mt-4 lg:inline-block lg:mt-0  text-black-200 font-medium hover:text-blue-800 mr-4">
+                        <a href="#" className="block mt-4 lg:inline-block lg:mt-0  text-black-200 font-medium hover:text-blue-800 mr-4">
                             Contact
                         </a>
                         </div>
                     <div className="pt-6">
-                        <a href="#" className="inline-block text-md px-4 py-2 leading-none border rounded text-black border-white hover:border-transparent hover:text-blue-800 hover:bg-white mt-4 lg:mt-0">Login</a>
-                        <a href="#" className="inline-block text-md px-4 py-2 leading-none border rounded text-white border-white bg-blue-500 hover:bg-blue-400 hover: mt-4 lg:mt-0 m-4">Sign Up</a>
+                        {user ?
+                        <><a href="/events" className="inline-block text-md px-4 py-2 mr-4 leading-none border rounded text-white bg-blue-500 border-white hover:border-transparent hover:text-white-800 hover:bg-blue-400 mt-4 lg:mt-0">Dashboard</a>
+                        <a href="/api/auth/logout" className="inline-block text-md px-4 py-2 mr-4 leading-none border rounded text-white bg-red-500 border-white hover:border-transparent hover:text-white-800 hover:bg-red-400 mt-4 lg:mt-0">Logout</a>
+                        </>
+                        :       
+                        <>                 
+                        <a href="/api/auth/login" className="inline-block text-md px-4 py-2 leading-none border rounded text-white border-white bg-green-500 hover:border-transparent hover:bg-green-400 mt-4 lg:mt-0">Login</a>
+                        <a href="/api/auth/signup" className="inline-block text-md px-4 py-2 leading-none border rounded text-white border-white bg-blue-500 hover:bg-blue-400 hover: mt-4 lg:mt-0 m-4">Sign Up</a>
+                        </>
+                        }
```

Comment:
Add `/* eslint-enable */` before the `}` here to re-enable the linter on the rest of the file. We just want to ignore the anchor tags.